src/components/DropMenu/DropMenu.less

```diff
@@ -0,0 +1,75 @@
+.lucid-DropMenu {
+
+	&.lucid-DropMenu-base {
+		display: inline-block;
+	}
+
+	.lucid-DropMenu-Control {
+		position:relative;
+		outline: 0px;
+	}
+
+	.lucid-DropMenu-Option {
+		cursor: pointer;
+		padding: 0 10px;
+		line-height: @Button-size-lineHeight;
+		font: @font;
+		height: 25px;
```

Comment:
Use same variable I mentioned in my [other comment](https://github.com/appnexus/lucid/pull/115/files#r56680215)